content/connect/mailing-lists.md

```diff
@@ -0,0 +1,9 @@
+---
```

Comment:
I think you want to move this file to content/connect/mailing-lists/index.md since this is not working: https://deploy-preview-661--jakartaee.netlify.com/connect/mailing-lists/